packages/@uppy/thumbnail-generator/src/index.js

```diff
@@ -186,9 +186,11 @@ module.exports = class ThumbnailGenerator extends Plugin {
       return this.createThumbnail(file, this.opts.thumbnailWidth)
         .then(preview => {
           this.setPreviewURL(file.id, preview)
+          this.uppy.emit('thumbnail:generated', this.uppy.getFile(file.id), preview)
         })
         .catch(err => {
           console.warn(err.stack || err.message)
+          this.uppy.emit('thumbnail:error', this.uppy.getFile(file.id), err)
```

Comment:
So `thumbnail:ready` is when all are ready, and `thumbnail:generated` is for single? Maybe add `all-` to it then, `thumbnail:all-ready` or `thumbnail:all-generated`?

Reply:
Had the ready event in a local file, was about to submit a PR to address #946, but this adds errors and all too, cool 😎